recipes/unbaffeld/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "unbaffeld" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/c4/a3/3dc6f9912137aab7da98640f90ac7ab314128c5c417d4f1b6321196ca0e5/UNBAFFELD-0.2.2.tar.gz"
+  sha256: ac032334a2ca1e5f1f783b12e96610b582ad2f59bc40af2a615cd25a8c54b120
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```

Comment:
I'm seeing other run time requirements in the `setup.py`:

```
requirements = [
    "inference-tools>=0.12.0",
    "numpy>=1.8",
    "scipy>=1.6.3",
    "h5py",
    "matplotlib>=3.4.2"
]
```

NOTE: you probably want to make the dependency `matplotlib-base` unless you have need for the QT gui.

Reply:
Done.